src/components/App/AppSagas.js

```diff
@@ -41,7 +44,8 @@ export function* appInit() {
     console.info('User is signed in');
     yield loadApp();
   } else {
-    yield takeEvery(AUTH_SUCCESS_ACTION, loadApp),
-      yield setPageAction(PAGES_ENUM.LOGIN);
+    yield takeEvery(AUTH_SUCCESS_ACTION, loadApp);
+    // yield put(push('/'));
```

Comment:
Why adding commented code?

Reply:
So basically commented code replaces old code, but commenting it do not affect app. I found it interesting for figuring out why this happens later. Also if needed I can copy paste and then just change the route path.